src/index.html

```diff
@@ -5,9 +5,69 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="wrapper" data-qa="card">
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="main image"
+          class="card__image"
+        >
+
+        <div class="card__title">
+          <h2 class="card__title-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__title-description">
+            Product code: 195434
+          </p>
+        </div>
+      </div>
+
+      <div class="star">
+        <div class="star__content">
+
+          <div class="stars stars--4">
```

Comment:
```suggestion
        <div class="star__content">
          <div class="stars stars--4">
```
don't add indents between parent and child elements, 3'rd point in checklist